product_multi_price/migrations/11.0.1.2.0/post-migration.py

```diff
@@ -0,0 +1,9 @@
+#  Copyright 2020 Tecnativa - Ernesto Tejeda
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    group_id = env.ref('product_multi_price.group_show_multi_prices').id
```

Comment:
The idea here is not to duplicate code, but call the post_init_hook, and don't use openupgradelib, as it's not needed:
```python
from odoo.addons.product_multi_price.hooks import post_init_hook

def migrate(cr, version):
    post_init_hook(cr, False)
```